### PR TITLE
Change Parallelstore installation script to not move log folder for debian/ubuntu

### PR DESCRIPTION
**Pull request by abbas1902**
-Added a hotfix for installation script as it fails to properly move the log folders for ubuntu+debian (reverting from a previous pr)

### Submission Checklist

Please take the following actions before submitting this pull request.

* Fork your PR branch from the Toolkit "develop" branch (not main)
* Test all changes with pre-commit in a local branch [#](https://goo.gle/hpc-toolkit#development)
* Confirm that "make tests" passes all tests
* Add or modify unit tests to cover code changes
* Ensure that unit test coverage remains above 80%
* Update all applicable documentation
* Follow Cluster Toolkit Contribution guidelines [#](https://goo.gle/hpc-toolkit-contributing)

### CODE REVIEW

File: modules/file-system/parallelstore/scripts/install-daos-client.sh [hunk 1]
```diff
@@ -68,13 +68,13 @@ sed -i "s/#.*transport_config/transport_config/g" $daos_config
 sed -i "s/#.*allow_insecure:.*false/  allow_insecure: true/g" $daos_config
 sed -i "s/.*access_points.*/access_points: $access_points/g" $daos_config
 
-# Move agent log destination from /tmp/ (default) to /var/log/daos_agent/
-mkdir -p /var/log/daos_agent
-chown daos_agent:daos_agent /var/log/daos_agent
-sed -i "s/#.*log_file:.*/log_file: \/var\/log\/daos_agent\/daos_agent.log/g" $daos_config
-
 # Start service
+# TODO move back to main block one script issue affecting ubuntu is fixed
```
Comment (harshthakkar01) [inline modules/file-system/parallelstore/scripts/install-daos-client.sh:72]: Move this above line 74

Comment (abbas1902) [reply modules/file-system/parallelstore/scripts/install-daos-client.sh]: Done

File: modules/file-system/parallelstore/scripts/install-daos-client.sh [hunk 2]
```diff
@@ -68,13 +68,13 @@ sed -i "s/#.*transport_config/transport_config/g" $daos_config
...
 sed -i "s/.*access_points.*/access_points: $access_points/g" $daos_config
 
-# Move agent log destination from /tmp/ (default) to /var/log/daos_agent/
-mkdir -p /var/log/daos_agent
-chown daos_agent:daos_agent /var/log/daos_agent
-sed -i "s/#.*log_file:.*/log_file: \/var\/log\/daos_agent\/daos_agent.log/g" $daos_config
-
 # Start service
 if grep -q "ID=\"rocky\"" /etc/os-release && lsb_release -rs | grep -q "8\.[0-9]"; then
+	# Update script to change default log destination folder, after daos_agent user is supported in debian and ubuntu.
```
Comment (harshthakkar01) [inline modules/file-system/parallelstore/scripts/install-daos-client.sh:73]: nit: Add TODO: In the beginning.

**Comment by harshthakkar01** [comment]
Update PR title and description to reflect the changes.

**Review by harshthakkar01 — APPROVED** [review]

**Pull request merged**